mods/bucket/init.lua

```diff
@@ -210,7 +211,8 @@ bucket.register_liquid(
 	"default:lava_flowing",
 	"bucket:bucket_lava",
 	"bucket_lava.png",
-	"Lava Bucket"
+	"Lava Bucket",
+	{tools = 1}
```

Comment:
Shouldn't it be `{tool = 1}` instead of `{tools = 1}`?

Reply:
Thanks, I'll change that.